.azure-pipelines-ci/templates/test.yaml

```diff
@@ -0,0 +1,14 @@
+steps:
+- task: PowerShell@2
+  displayName: 'Test'
+  inputs:
+    targetType: inline
+    pwsh: ${{ parameters.pwsh }}
+    script: |
+      Import-Module .\tools\appveyor.psm1
+      Invoke-AppveyorTest -CheckoutPath $env:BUILD_SOURCESDIRECTORY
+- task: PublishTestResults@2
+  inputs:
+    testRunner: NUnit
+    testResultsFiles: 'TestResults.xml'
+    condition: succeededOrFailed()
```

Comment:
Something that should be said is that although I set it to always run, even it fails in order to upload test results, this task does not run if there is a test failure unfortunately. This seems to be a bug with the Azure DevOps YAML though. We should raise something but I don't think that's a blocking issue

Reply:
@JamesWTruher FYI